apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -1097,4 +1098,89 @@ export class BookingsService_2024_08_13 {
       t: await getTranslation("en", "common"),
     });
   }
+
+  async getBookingsCount(
```

Comment:
Can we move the newly added functions to `BookingsStatisticsService` ? if we keep adding more and more in this service will be hard to understand whats going on.